synapse/models/infotech.py

```diff
@@ -956,6 +966,28 @@ def getModelDefs(self):
                         'doc': 'The most recent version of the rule evaluated as a match.'}),
                 )),
 
+                ('it:reveng:function', {}, ()),
+
+                ('it:reveng:file:function', {}, (
+                    ('function', ('it:reveng:function', {}), {
+                        'ro': True,
+                        'doc': 'The guid matching the function'}),
+                    ('file', ('file:bytes', {}), {
+                        'ro': True,
+                        'doc': 'The file that contains the function.'}),
+                    ('va', ('int', {}), {
```

Comment:
is this secondary property mutable within a given file?

Reply:
also, can a given function occur more than once in a given file?

Reply:
It's optional. You don't need to add it when first making an `it:reveng:file:function` node, but it is useful for tracking function locations in the file. But also a code block is an interesting concept when you take into account function thunks and hydra functions where "first" could mean slightly different things. So I just left it editable.

Reply:
for now, we dont care about a symbolic function occuring at more than one rva in a given file.  The only other option would be to include rva in the comp, which would preclude creating them if you dont know the rva of the function. :)